internal/sdkv2provider/schema_cloudflare_notification_policy.go

```diff
@@ -59,6 +59,7 @@ func resourceCloudflareNotificationPolicySchema() map[string]*schema.Schema {
 				"http_alert_origin_error",
 				"load_balancing_health_alert",
 				"load_balancing_pool_enablement_alert",
+				"pages_event_alert",
```

Comment:
can you link me to the public docs for this one? i'm struggling to find it.

Reply:
Listed on the [notification docs](https://developers.cloudflare.com/fundamentals/notifications/notification-available/#pages)

The actual value I got from a worker I have which checks all possible alert types https://api.cyberjake.xyz/cloudflare_api/alert_types and search for Pages.

Reply:
hmm, this looks like the old issue of telling us what is available but isn't in the api-schemas. let me chase the internal teams.